test/calendar.js

```diff
@@ -1,6 +1,7 @@
 const test = require('tape')
 const Calendar = require('../src/calendar.js')
 const Utils = require('../src/utils.js')
+const { URL } = require('url')
```

Comment:
Maybe it is auto merging stuff that add URL here and in the src/calendar.js file